### ISSUE TITLE
Commenting indented blocks does not respect `use_consistent_indentation` on blank lines

### ISSUE THREAD

**Issue by YodaEmbedding** [b3nj5m1n/kommentary#26]
Attempting to comment an indented block does not respect `use_consistent_indentation = true`.

Test case:

```python
def func(x):
    def inner(x):
        print(x)

        raise NotImplementedError
```

Expected:

```python
def func(x):
    # def inner(x):
    #     print(x)
    #
    #     raise NotImplementedError
```

Actual:

```python
def func(x):
    # def inner(x):
    #     print(x)
#
    #     raise NotImplementedError
```

----

A similar issue: attempting to uncomment the "expected" version also leaves trailing whitespace on the blank lines.

**Comment by b3nj5m1n** [comment]
Should be fixed with def5488e25af0d46791d8e1cbe86a52f82d3f0af, let me know if you still have problems.

**Comment by YodaEmbedding** [comment]
Looks good.
